webpack.config.js

```diff
@@ -26,6 +26,30 @@ module.exports = async (env, options) => {
       taskpane: ['react-hot-loader/patch', './src/taskpane/index.tsx'],
       commands: './src/commands/commands.ts',
     },
+    optimization: {
+      splitChunks: {
+        chunks: 'all',
+        minSize: 20000,
+        maxSize: 50000,
+        minChunks: 1,
+        maxAsyncRequests: 30,
+        maxInitialRequests: 30,
+        automaticNameDelimiter: '~',
+        enforceSizeThreshold: 50000,
+        cacheGroups: {
```

Comment:
Most of these seem to be the default values. I think its more maintainable if only non-default values are specified, or is it necessary to specify everything?